src/System.Runtime/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -1,8 +1 @@
-Compat issues with assembly System.Runtime:
-MembersMustExist : Member 'System.GC.GetAllocatedBytesForCurrentThread()' does not exist in the implementation but it does exist in the contract.
```

Comment:
It would be nice if there was a build error when there were unnecessary baseline entries, so they don't have to be cleaned up periodically like this.

Reply:
Yes it would be a nice feature to at least warn about these. Perhaps a file a request on BuildTools to add that.

Reply:
Filed https://github.com/dotnet/buildtools/issues/1851